addons/mass_mailing/static/src/themes/theme_selector/theme_selector.scss

```diff
@@ -115,27 +114,51 @@
             }
         }
     }
+    .o_mail_templates_grid {
+        display: grid;
+        grid-template-columns: repeat(5, 20%);
+        grid-auto-rows: 1fr;
+    }
 
     .o_mail_template_preview {
         color: #ffffff;
         cursor: pointer;
         padding: 10px 1.5rem;
-        width: 20%;
-        div i.o_mail_template_remove_favorite {
-            display: none;
+        img {
+            width: 100%;
+            height: 100%;
         }
-        div:hover i.o_mail_template_remove_favorite {
-            display: inline-block;
+        i.fa-star {
+            color: $o-main-favorite-color;
+        }
+        .o_mail_template_remove_favorite i {
+            display:  none;
+        }
+        &:hover .o_mail_template_remove_favorite i {            
+            display: block;
             &:hover {
                 color: red;
             }
         }
-        img {
-            width: 20px;
-            height: 20px;
-        }
-        i.fa-star {
-            color: $o-main-favorite-color;
+    }
+
+    .o_mail_favorite_preview_wrapper {
+        display: inline-block;
+        width: 100%;
+        position: relative;
+        flex-grow: 1;
+        background-color: #e8ebee;
```

Comment:
was this specific color required? If not, shouldn't we put transparent instead? (not really important, but hardcoded colors are not common)